app/components/NavModal/index.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+const NavModalDialog = styled.div.attrs({
+  className: 'modal-dialog',
+})`
+  &&& {
+    font-size: 2rem;
+    font-weight: 600;
+  }
+`;
+
+const NavModalContent = styled.div.attrs({
+  className: 'modal-content',
```

Comment:
You prefer this to `<NavModalContent className="modal-content">`?

I think I prefer consistency here -- would prefer to keep all classNames in the JSX

Reply:
Just replying so there's a record saying you already got back to me and are OK with this.